src/Components/CreateImageWizard/formComponents/ReviewStepTables.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+
+import { Panel, PanelMain } from '@patternfly/react-core';
+import {
+  TableComposable,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  Tr,
+} from '@patternfly/react-table';
+import PropTypes from 'prop-types';
+
+import { UNIT_GIB, UNIT_MIB } from '../../../constants';
+
+export const FSReviewTable = ({ ...props }) => {
+  return (
+    <Panel isScrollable>
+      <PanelMain maxHeight="30ch">
+        <TableComposable
+          aria-label="File system configuration table"
+          variant="compact"
+        >
+          <Thead>
+            <Tr>
+              <Th>Mount point</Th>
+              <Th>File system type</Th>
+              <Th>Minimum size</Th>
+            </Tr>
+          </Thead>
+          <Tbody data-testid="file-system-configuration-tbody-review">
+            {props.fsc.map((r, ri) => (
+              <Tr key={ri}>
+                <Td className="pf-m-width-30">{r.mountpoint}</Td>
+                <Td className="pf-m-width-30">xfs</Td>
+                <Td className="pf-m-width-30">
+                  {r.size}{' '}
+                  {r.unit === UNIT_GIB
+                    ? 'GiB'
+                    : r.unit === UNIT_MIB
+                    ? 'MiB'
+                    : 'KiB'}
+                </Td>
+              </Tr>
+            ))}
+          </Tbody>
+        </TableComposable>
+      </PanelMain>
+    </Panel>
+  );
+};
+
+export const PackagesTable = ({ ...props }) => {
+  return (
+    <Panel isScrollable>
+      <PanelMain maxHeight="30ch">
+        <TableComposable aria-label="Packages table" variant="compact">
+          <Thead>
+            <Tr>
+              <Th>Name</Th>
+            </Tr>
+          </Thead>
+          <Tbody data-testid="packages-tbody-review">
+            {props.packages.map((r, ri) => (
+              <Tr key={ri}>
+                <Td className="pf-m-width-30">{r.name}</Td>
+              </Tr>
+            ))}
+          </Tbody>
+        </TableComposable>
+      </PanelMain>
+    </Panel>
+  );
+};
+
+export const RepositoriesTable = ({ ...props }) => {
```

Comment:
Could we get rid of the props by calling `const repositories={getState()?.values?.['custom-repositories']}` inside this component?